tests/regression/02-base/49-unknown_func_union.c

```diff
@@ -14,10 +14,7 @@ typedef union either {
     struct list node;
 } either_t;
 
-// void mutate_either(either_t e){
-//     list_t *next = e.node.next;
-//     next->val = 42;
-// }
```

Comment:
Maybe leave this commented out code here, such that one can understand why this invalidation is needed?